mkosi/distributions/fedora.py

```diff
@@ -14,7 +16,33 @@
 from mkosi.installer.dnf import Dnf
 from mkosi.installer.rpm import RpmRepository, find_rpm_gpgkey, setup_rpm
 from mkosi.log import die
-from mkosi.util import listify
+from mkosi.util import listify, tuplify
+
+
+@tuplify
+def find_fedora_rpm_gpgkeys(context: Context) -> Iterable[str]:
+    key1 = find_rpm_gpgkey(context, key=f"RPM-GPG-KEY-fedora-{context.config.release}-primary")
+    key2 = find_rpm_gpgkey(context, key=f"RPM-GPG-KEY-fedora-{context.config.release}-secondary")
+
+    if key1:
+        # During branching, there is always a kerfuffle with the key transition.
+        # For Rawhide, try to load the N+1 key, just in case our local configuration
+        # still indicates that Rawhide==N, but really Rawhide==N=1.
+        if context.config.release == "rawhide" and key1.startswith("file://"):
+            path = Path(key1[7:]).resolve()
```

Comment:
I think
```suggestion
            path = Path(key1.removeprefix("file://")).resolve()
```
documents the intent more clearly than the `[7:]` and we don't need the performance here.

Reply:
It'd be great if `removeprefix` returned None, because then one could collapse the whole thing into the conditional. Unfortunately they chose a different interface.